client/components/Assignments/AssignmentGroupList.jsx

```diff
@@ -119,7 +120,7 @@ class AssignmentGroupListComponent extends React.Component {
                             className="sd-list-header__name sd-list-header__name--cursorPointer"
                             onClick={this.changeAssignmentListSingleGroupView}
                         >
-                            <span>{ASSIGNMENTS.LIST_GROUPS[groupKey].label}</span>
+                            <span>{gettext(ASSIGNMENTS.LIST_GROUPS[groupKey].label)}</span>
```

Comment:
`gettext` is already called here https://github.com/superdesk/superdesk-planning/pull/889/files#diff-c90901640fd209305b37428d71c2dcafR68
Do we need to call `gettext` twice?

Reply:
yes, problem is that when it's defined on top level as a constant it won't have the translations loaded yet so it will just return the same string. so the translation will only happen when rendering. but we have to keep the gettext there for string extraction